_includes/manuals/nightly/3.6_upgrade.md

```diff
@@ -95,6 +95,12 @@ If the Ruby 2.5 module was previously enabled, it must be reset:
 dnf -y module reset ruby
 {% endhighlight %}
 
+Switch to the PostgreSQL 13 module:
+
+{% highlight bash %}
+dnf -y module switch-to postgresql:13
```

Comment:
Wait, do we need to add `switch-to` in the web UI too?

Reply:
Web UI?

Reply:
![image](https://github.com/theforeman/theforeman.org/assets/22042343/b2ce05a8-43da-483c-9bd7-1fc7623b65ab)

Reply:
I know it's not related to these changes, but we have module stream actions available on the host details page.

Reply:
Ah, yes. Then I'd add a switch to option with a selection of which version you want to switch to (like enable)

Reply:
You mean as a Katello feature for host actions?

Reply:
Yes, for host actions (sorry for the tangentially related comments)

Reply:
> I know it's not related to these changes

This is often how you learn about features: just looking at changes from others. Please keep it up.